app.py

```diff
@@ -14,7 +14,7 @@
 app = Flask(__name__)
 github_app = GitHubApp(app)
 directory = DirectoryClient()
-addUserAsMember = os.environ.get("ADD_MEMBER", False)
+addUserAsMember = os.getenv('ADD_MEMBER', 'False') == 'True'
```

Comment:
I didn't catch this in the last PR, but this should go in `githubapp/__init__.py` instead of here

Reply:
What about removing `if org.is_member(user)` - or it is better with env variable specifying if the user should be added to the org?

Reply:
@kalisb it's better to keep it as a variable. There are cases when you _don't_ want to add people to an org, like when you're using GHEC and accidentally fat-finger a user ID, or if you're using a backend like LDAP where a user's email is tied to a different user account on github.com. It should be an opt-in behavior